hw/dv/tools/xcelium/common.ccf

```diff
@@ -34,7 +34,7 @@ set_statement_scoring
 set_toggle_scoring -sv_enum enable_mda -sv_struct_with_enum -sv_modport -sv_mda 16 -sv_mda_of_struct -sv_generate -sv_packed_union
 
 // Enables scoring of reset states and transitions for identified FSMs.
-set_fsm_reset_scoring
+// set_fsm_reset_scoring
```

Comment:
nit: perhaps remove these lines, we probably don't need them any more since Lint will cover this.